core/src/main/java/nl/nn/adapterframework/extensions/javascript/J2V8.java

```diff
@@ -107,4 +114,20 @@ public Object invoke(V8Object receiver, V8Array parameters) {
 			}
 		}, sender.getName());
 	}
+
+	@Override
+	public void setResultHandler(ResultHandler resultHandler) {
+		getEngine().registerJavaMethod(new JavaVoidCallback() {
+			@Override
+			public void invoke(V8Object receiver, V8Array parameters) {
+				resultHandler.setResult(parameters.getString(0));
+			}
+		}, "result");
+		getEngine().registerJavaMethod(new JavaVoidCallback() {
+			@Override
+			public void invoke(V8Object receiver, V8Array parameters) {
+				resultHandler.setError(parameters.getString(0));
+			}
+		}, "error");
+	}
```

Comment:
You can't just move this here? the resulthandler (result and error) are very specific to graphviz and this class is also used in pipes/senders...